org.eclipse.xtext.xbase.lib/build.gradle

```diff
@@ -5,3 +5,23 @@ description = 'The runtime library for Xbase languages.'
 dependencies {
 	compile "com.google.guava:guava:$versions.guava"
 }
+
+task copyGwtSources(dependsOn: 'build') {
+	doLast {
+	
+		ant.copy(toDir: '../org.eclipse.xtext.xbase.lib.gwt/build/gwtSources', verbose: true) {
```

Comment:
the original one did a copy with filter only but that does not seem to work with gradle ant builder, at least i did not get it running. => i did the copy & delete both in ant